### PR TITLE
[VOY-82] Creation of Currency and ExchangeRate models

### PR DESCRIPTION
**Pull request by rdemasi**

### CODE REVIEW

**Comment by arieliten** [comment]
Task linked: [VOY-82 Crear modelos de acuerdo al ERD acordado](https://app.clickup.com/t/6647153/VOY-82)

File: db/seeds/currencies.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+CURRENCIES = [
+  { code: 'ARS', name: 'Argentine Peso' },
+  { code: 'USD', name: 'US Dollar' },
+  { code: 'EUR', name: 'Euro' },
+  { code: 'CAD', name: 'Canadian Dollar' },
+  { code: 'GBP', name: 'Pound Sterling' },
+  { code: 'BRL', name: 'Brazilian Real' },
+  { code: 'CLP', name: 'Chilean Peso' }
+].freeze
```
Comment (arieliten) [inline db/seeds/currencies.rb:11]: I don't think we need so many currencies. Also I thought we agreed on not creating the model currency and instead use static values (constants) in the code for that. But it's ok for now if you already added the model. We can revisit and remove it later if we find that's not needed

**Review by arieliten — APPROVED** [review]
A few minor things, but overall looks good 🚀

File: db/schema.rb [hunk 2]
```diff
@@ -33,6 +33,32 @@
...
+    t.datetime "updated_at", precision: 6, null: false
+    t.index ["area_id"], name: "index_budget_estimations_on_area_id"
+    t.index ["book_account_id"], name: "index_budget_estimations_on_book_account_id"
+    t.index ["bussines_unit_id"], name: "index_budget_estimations_on_bussines_unit_id"
+    t.index ["category_id"], name: "index_budget_estimations_on_category_id"
+    t.index ["company_id"], name: "index_budget_estimations_on_company_id"
+    t.index ["created_by_id"], name: "index_budget_estimations_on_created_by_id"
+    t.index ["deleted_by_id"], name: "index_budget_estimations_on_deleted_by_id"
+    t.index ["month", "year"], name: "index_budget_estimations_on_month_and_year", unique: true
+  end
```
[the remark this answers is not in the document]
Comment (rdemasi) [reply db/schema.rb]: Removed it ;-)

**Pull request merged**